server/routes/manageConditions/index.ts

```diff
@@ -73,30 +72,18 @@ export default function Index({ licenceService, conditionService }: Services): R
   }
 
   // START MEZ
-  // remove area from map list in form builder without confirmation (delete single conditions) - DELETE NOT USED
-  {
-    const controller = new AdditionalLicenceConditionRemoveUploadRoutes(licenceService)
-    get('/condition/id/:conditionId/remove-upload', controller.GET)
-  }
```

Comment:
Removed this as it wasn't being used - only reference was setting a variable that was not used